medusa/search/proper.py

```diff
@@ -186,12 +186,16 @@ def _get_proper_results(self):  # pylint: disable=too-many-locals, too-many-bran
         sorted_propers = sorted(propers.values(), key=operator.attrgetter('date'), reverse=True)
         final_propers = []
 
-        # Keep only last 100 items of processed propers:
-        self.processed_propers = self.processed_propers[-100:]
+        # Keep only items from last 3 days in processed propers:
+        latest_proper = datetime.datetime.now() - datetime.timedelta(days=3)
```

Comment:
@p0psicles this is a new setting. the PROPER_SEARCH_DAYS (used for search)

this setting is to CLEAN the already_processed  propers we store to avoid to guessit over and over.